kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -42,9 +42,9 @@ def build(self):
 
         # check the shape
         if isinstance(self.shape, numbers.Integral):
-            self.T = self.shape
+            self.N = self.shape
         elif isinstance(self.shape, tuple):
-            self.T = self.shape[0]
+            self.N = self.shape[0]
```

Comment:
so self.N is basically an alias for self.shape[0] throughout the library?

Reply:
Something like that. Would be good to investigate this further. Can you make an issue?

Reply:
ok